magstock/configspec.ini

```diff
@@ -1,2 +1,2 @@
-
-
+# The cost of a food wristband, which entitles attendees to meals provided by the event.
+food_price = integer(default='20')
```

Comment:
Minor style note: It would be more idiomatic to say ``integer(default=20)`` without the quotes, though the end result is the same.